src/Controller/Api/AdherentsController.php

```diff
@@ -17,30 +17,35 @@
 class AdherentsController extends Controller
 {
     /**
-     * @Route("/count", name="app_referent_dashboard_users")
+     * @Route("/count", name="app_adherents_count")
      * @Method("GET")
      */
-    public function usersAction(AdherentRepository $adherentRepository): Response
+    public function adherentsCountAction(AdherentRepository $adherentRepository): Response
     {
         $count = $adherentRepository->countByGender();
 
-        array_walk($count, function (&$item) {
-            $item = (int) $item['count'];
-        });
-
-        $total = array_sum($count);
-
-        array_walk($count, function (&$item) use ($total) {
-            $item = $this->calculatePercentage($item, $total);
-        });
+        return new JsonResponse($this->aggregateCount($count));
+    }
 
-        $count['total'] = $total;
+    /**
+     * @Route("/count_for_referent_managed_area", name="app_adherents_count_for_referent_managed_area")
```

Comment:
cs: hyphen case and I do thing the route should be called `count-by-referent-area`

Sorry I missed that on first review.